libr/core/cmd.c

```diff
@@ -270,9 +270,9 @@ static const RCoreHelpMessage help_msg_equalh = {
 	"=h-", "", "stop background webserver",
 	"=h--", "", "stop foreground webserver",
 	"=h*", "", "restart current webserver",
-	"=h&", " port", "start http server in background",
-	"=H", " port", "launch browser and listen for http",
-	"=H&", " port", "launch browser and listen for http in background",
+	"=h&", " port", "start http server on 'port' in background",
+	"=H", " port", "listen for http on 'port' (launch browser)",
```

Comment:
Maybe use “start http server” is more correct than listen for http

Reply:
certainly